src/main/resources/application.yaml

```diff
@@ -35,3 +35,5 @@ sftp:
 sscs.gaps2.schema.location:
   delta: /schema/SSCS_Extract_Schema_0.8.xsd
   reference: /schema/SSCS_Extract_Schema_Reference_0.2.xsd
```

Comment:
sscs.gaps2.schema.location Is this still needed?. I think the code was moved to enum.

Reply:
yes, that squeezed in by mistake. I fix it.

Reply:
done. Thanks, @sathishbabuj.